test/powershell/Modules/Microsoft.PowerShell.Management/Start-Process.Tests.ps1

```diff
@@ -1,86 +1,120 @@
+	    $process = Start-Process ping -WorkingDirectory $pingDirectory -ArgumentList $pingParam -PassThru -RedirectStandardOutput "$TESTDRIVE/output"
 
-	$process.Length      | Should Be 1
-	$process.Id          | Should BeGreaterThan 1
-	# $process.ProcessName | Should Be "ping"
+	    $process.Length      | Should Be 1
+	    $process.Id          | Should BeGreaterThan 1
+	    # $process.ProcessName | Should Be "ping"
     }
 
-    It "Should should handle stderr redirection without error" {
-	$process = Start-Process ping -ArgumentList $pingParam -PassThru -RedirectStandardError $tempFile -RedirectStandardOutput "$TESTDRIVE/output"
+    It "Should handle stderr redirection without error" {
+	    $process = Start-Process ping -ArgumentList $pingParam -PassThru -RedirectStandardError $tempFile -RedirectStandardOutput "$TESTDRIVE/output"
 
-	$process.Length      | Should Be 1
-	$process.Id          | Should BeGreaterThan 1
-	# $process.ProcessName | Should Be "ping"
+	    $process.Length      | Should Be 1
+	    $process.Id          | Should BeGreaterThan 1
+	    # $process.ProcessName | Should Be "ping"
     }
 
-    It "Should should handle stdout redirection without error" {
-	$process = Start-Process ping -ArgumentList $pingParam -Wait -RedirectStandardOutput $tempFile
-	$dirEntry = get-childitem $tempFile
-	$dirEntry.Length | Should BeGreaterThan 0
+    It "Should handle stdout redirection without error" {
+	    $process = Start-Process ping -ArgumentList $pingParam -Wait -RedirectStandardOutput $tempFile
+	    $dirEntry = get-childitem $tempFile
+	    $dirEntry.Length | Should BeGreaterThan 0
     }
 
     # Marking this test 'pending' to unblock daily builds. Filed issue : https://github.com/PowerShell/PowerShell/issues/2396
-    It "Should should handle stdin redirection without error" -Pending {
-	$process = Start-Process sort -Wait -RedirectStandardOutput $tempFile -RedirectStandardInput $assetsFile
-	$dirEntry = get-childitem $tempFile
-	$dirEntry.Length | Should BeGreaterThan 0
+    It "Should handle stdin redirection without error" -Pending {
+	    $process = Start-Process sort -Wait -RedirectStandardOutput $tempFile -RedirectStandardInput $assetsFile
+	    $dirEntry = get-childitem $tempFile
+	    $dirEntry.Length | Should BeGreaterThan 0
     }
 
-    It "Should give an error when Verb parameter is used" -Skip:(-not $IsCoreClr) {
-        try
-        {
-            Start-Process -Verb runas -FilePath $pingCommand -ArgumentList $pingParam
-            throw "No Exception!"
-        }
-        catch
-        {
-            $_.FullyQualifiedErrorId | Should be "NotSupportedException,Microsoft.PowerShell.Commands.StartProcessCommand"
-            $_.Exception.Message | Should match '-Verb'
-        }
+    ## -Verb is supported in PowerShell core on Windows full desktop.
+    It "Should give an error when -Verb parameter is used" -Skip:$isFullWin {
```

Comment:
No, `-Verb` is not supported on non-windows platforms and windows headless SKUs. So on NanoServer and IoT, using `-Verb` should result in a terminating error.